modules/document_repository/test/TestPlan.md

```diff
@@ -1,5 +1,7 @@
-1. User has access to document repository if they have the "View and upload files in Document Repository” permission or is the super user.
-   [Automation Testing]
```

Comment:
This isn't in the integration tests. Only super user is checked, which is not a very useful test since super user will always have access.